multi-node/aws/cmd/kube-aws/command_validate.go

```diff
@@ -54,8 +53,10 @@ func runCmdValidate(cmd *cobra.Command, args []string) error {
 	}
 
 	if err != nil {
-		return fmt.Errorf("Error creating cluster: %v", err)
```

Comment:
This was just leading to output like "Error creating cluster: Error: Key does not exist".  I think the "Validating stack template..." output added in the validate command makes the extra messaging unnecessary.